### DISCUSSION TITLE
感谢作者对开源的分享，打赏一下支持作者

### DISCUSSION THREAD

**Discussion by BoyFaceGirl** [iNKORE-NET/UI.WPF.Modern#345] [General]
![image](https://github.com/InkoreStudios/UI.WPF.Modern/assets/8773178/2e6d2d4c-3687-4909-8361-b37fc8902a81)

![image](https://github.com/InkoreStudios/UI.WPF.Modern/assets/8773178/9029ad5c-19bb-460c-b48d-cd6a0db282ed)

**Comment by NotYoojun** [comment]
感谢支持！这是我的赞助页面：
https://inkore.net/zh-cn/about/members/notyoojun#sponsor

**Comment by BoyFaceGirl** [comment]
> 感谢支持！这是我的赞助页面： https://inkore.net/zh-cn/about/members/notyoojun#sponsor

建议挂到read.md 里面

**Comment by NotYoojun** [comment]
好的，加了，谢谢
